hamster.js

```diff
@@ -2,24 +2,26 @@ var request = require('request');
 var open = require('open');
 
 // 1. Find the correct URL to get adorable hamsters from Giphy!
-request('GIPHY API GET REQUEST URL', function (error, response, body) {
+request('dc6zaTOxFJmzC', function (error, response, body) {
```

Comment:
In the giphy documentation you needed to look at the search endpoint to see the link. It should be: `http://api.giphy.com/v1/gifs/search?q=funny+cat&api_key=dc6zaTOxFJmzC `